config/kafka.php

```diff
@@ -84,7 +86,7 @@
 
     'middlewares' => [
         'consumer' => [
-            \Metamorphosis\Middlewares\Log::class,
+            Log::class,
```

Comment:
on `config` folder, I think that this looks odd. what is your opinion about it?

Reply:
yes. this rule should only apply for the `src/` directory

Reply:
`tests/` also

Reply:
👍 positive